response/response.go

```diff
@@ -62,11 +64,19 @@ func (res *Response) Cookies(cs ...*cookie.Cookie) *Response {
 	return res
 }
 
+func (res *Response) Keep(keepFunc func(w http.ResponseWriter)) *Response {
```

Comment:
I think `Keep` can't explain what this function doing. maybe just call stream?
and I will be thinking we shouldn't put this in `Response`, we should create a `Stream` just like `File`.
because of use `response.New("").Keep(func() {....})` is little odd, the user may not expected we will send `""` before `Keep` function work.